lib/src/connector/score_connector.dart

```diff
@@ -141,13 +135,18 @@ class ScoreConnector {
       result = await Connector.getDataByGet(parameter);
       tagNode = parse(result);
       rankNodes = tagNode.getElementsByTagName("tbody")[0].getElementsByTagName("tr");
-      rankNodes = rankNodes.getRange(2, rankNodes.length).toList().reversed.toList();
+      final List<Element> filteredRankNodes = [];
+      for (final rankNode in rankNodes) {
+        if (rankNode.getElementsByTagName("td").length < 7) continue;
```

Comment:
```suggestion
        final rankNodeColumns = rankNode.getElementsByTagName("td");
        if (rankNodeColumns.length < 7) continue;
```

Suggest always simplify the operations within 'if' structures as much as possible, as complex steps may make debugging more difficult and are more likely to hide unforeseen issues.
In this example, we could separate out the '`getElementsByTagName`' step so that it stands alone in its own line.

Reply:
i modified the .where() method instead to get the filtered rank nodes with tr element count greater than or equal to 7 in commit https://github.com/NEO-TAT/tat_flutter/commit/9325e4f3cd8d5c53d4adefc380817dac53e1e7b1

Reply:
ca584f88d7d5c2bb6a9de9a4aed25a3deea0a093